README.md

```diff
@@ -75,9 +75,20 @@ Then use `ninja -C out/Release` or `ninja -C out/Debug` to build WebNN-native.
 
 ### Run tests
 
-Run unit tests, for example `./out/Release/webnn_unittests`.
+Run unit tests:
+```sh
+> ./out/Release/webnn_unittests
+```
 
-Run end2end tests, for example `./out/Release/webnn_end2end_tests`.
+Run end2end tests on a Default device:
+```sh
+> ./out/Release/webnn_end2end_tests
+```
+You can also specify a device to run end2end tests using "-d" option, for example:
+```sh
+> ./out/Release/webnn_end2end_tests -d GPU
+```
+Currently "CPU", "GPU" and "Default" are supported, more devices are to be supported in the future.
```

Comment:
@mingmingtasd What does “Default” mean? Thanks.

Reply:
If set -d Default or without -d option, tests will run on a default device.
Default represents different devices on different backends, for example:
GPU is the default device on DML, while CPU is the default device on IE. @BruceDai

Reply:
@mingmingtasd  It's clear for me, thanks much for your explanation.